tracing-opentelemetry/src/lib.rs

```diff
@@ -30,6 +30,10 @@
 //!     let root = span!(tracing::Level::TRACE, "app_start", work_units = 2);
 //!     let _enter = root.enter();
 //!
+//!     // Optionally set a dynamic span name for otel to export
+//!     let operation = "GET http://example.com".to_string();
+//!     span!(tracing::Level::TRACE, "http_request", otel.name=operation.as_str());
```

Comment:
If we expect there to ever be other fields with special meaning to OpenTelemetry namespaced under `otel.`, should we consider having a master list in the docs, as well?

Reply:
hm yeah could be a section in the docs

Reply:
Added 👍